Source/Common/ContextJsonConverter.cs

```diff
@@ -11,46 +11,34 @@ namespace Schema.NET
     public class ContextJsonConverter : JsonConverter<JsonLdContext>
     {
         /// <inheritdoc />
-        public override JsonLdContext ReadJson(JsonReader reader, Type objectType, JsonLdContext? existingValue, bool hasExistingValue, JsonSerializer serializer)
+        public override JsonLdContext Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
-            if (reader is null)
+            if (typeToConvert is null)
```

Comment:
We should null check all parameters.

Reply:
We don't actually use the `options` parameter - would it still be worth adding checks?

Reply:
In that case, I guess we can leave it.